primes.rb

```diff
@@ -1,6 +1,19 @@
+######CANT LAUNCH SERVER WITHOUT THIS CODE###########
 class Primes
-  def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
-  end
-end
\ No newline at end of file
+  def self.sum_to(limit)
+  # TODO - add your prime number solution here...
+    require 'prime'
+####################################################
+def self.sum_to(n)
+   (2..n/2).none?{ |x| n % x == 0 }
+end
+
+primes = []
+for i in 2...1000
+   primes.push(i) if self.sum_to(i)
+end
+#Prints the sum of the range - NOTE TO SELF - look into "inject" method more.
+primes.inject(:+)
+ end
+
+end
```

Comment:
I fixed this and completely rewrote the code, looks better, works better - no bug on refresh. Not sure how I got it so wrong in the first place! Pushing changes now but it's taking a while :(